docs/src/.overrides/assets/stylesheets/extra.css

```diff
@@ -33,7 +63,7 @@
 
     /* footer */
     /* previous/next text */
-    --md-footer-fg-color: var(--dj-primary);
+    /* --md-footer-fg-color: var(--dj-primary); */
```

Comment:
Is this commented out here? Preserved in case we change our minds down the road?

Reply:
Yes, it is commented in case we want to use the footer in the future.